octoprint/umbrel-app.yml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: octoprint
+category: Networking
+name: OctoPrint
+version: "1.8.7"
+tagline: The snappy web interface for your 3D printer.
+description: >-
+  OctoPrint provides a snappy web interface for controlling consumer 3D printers.
+releaseNotes: >-
+  Improvements
+   Add support for Python 3.11 by bumping up the version of a dependency.
+  Bug fixes
+    Do not send None to Locale.parse if no match can be made between the request's accepted languages and the installed languages. Fixes a 500 server error on the API in case of Babel being at of version of 2.12 or higher.
```

Comment:
For the initial app release we can either remove `releaseNotes` entirely, or else just say:
` - Initial release for the Umbrel App Store`

Reply:
done